src/averageAges.js

```diff
@@ -15,11 +15,27 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let count = 0;
+
+  const mensArr = people.filter((person) => {
+    return person.sex === 'm';
+  });
+
+  const sumAge = mensArr.reduce((prev, person) => {
+    const personDiedCentury = Math.ceil(person.died / 100);
+    const age = person.died - person.born;
+    const checking = personDiedCentury === century || !century;
+
+    if (checking) {
```

Comment:
Replace the `if` statement with a ternary operator for better readability. Example: `return checking ? prev + age : prev;`